### ISSUE TITLE
django-constance and django-modeltranslation compatibility

### ISSUE THREAD

**Issue by SilmarillionUA** [jazzband/django-constance#281]
Good day, is there a way to make django-constance work with [django-modeltranslation](https://github.com/deschler/django-modeltranslation)?

**Comment by camilonova** [comment]
You can translate the variable names using `_` at your settings, just like any other string translation.

**Comment by parruc** [comment]
I'm having the exact same question but I guess @SilmarillionUA was not referring to the variable names but instead to the constance contents.

It would help to be able to mark a certain constance field to be "translatable" and have, as a result, one field to feel inside constance admin area for each available language in the website.

**Comment by zuxriddin99** [comment]
> I'm having the exact same question but I guess @SilmarillionUA was not referring to the variable names but instead to the constance contents.
> 
> It would help to be able to mark a certain constance field to be "translatable" and have, as a result, one field to feel inside constance admin area for each available language in the website.

please can you explain with an example?

**Comment by parruc** [comment]
Let's say in my settings I have:

`LANGUAGES = ["en", "it"]
CONSTANCE_ADDITIONAL_FIELDS = {
    'richtext': ['django.forms.CharField', {
        'widget': 'tinymce.widgets.TinyMCE',
    }],
CONSTANCE_CONFIG = OrderedDict([
    ('COOKIESPOLICY_TEXT', ("<h1>Cookie policy</h1>", "Cookies text", "richtext")),
    ('PRIVACYPOLICY_TEXT', ("<h1>Privacy policy</h1>", "Privacy text", "richtext"))
])`

It would be cool to have some way to tell constance "hey my COOKIESPOLICY constance config is translatable". 
If this is the case I will have one field to fill in django admin for each active langauge and I will be able to fill different values for different languages.
The bottom line is that I will have the correct cookiespolicy language shown in my website based on the active language.
I hope to have been clear in my messy explanation ;-)

**Comment by zuxriddin99** [comment]
> Let's say in my settings I have:
> 
> ```python
> LANGUAGES` = ["en", "it"]
> CONSTANCE_ADDITIONAL_FIELDS = {
>     'richtext': ['django.forms.CharField', {
>         'widget': 'tinymce.widgets.TinyMCE',
>     }],
> CONSTANCE_CONFIG = OrderedDict([
>     ('COOKIESPOLICY_TEXT', ("<h1>Cookie policy</h1>", "Cookies text", "richtext")),
>     ('PRIVACYPOLICY_TEXT', ("<h1>Privacy policy</h1>", "Privacy text", "richtext"))
> ])
> ```
> 
> It would be cool to have some way to tell constance "hey my COOKIESPOLICY constance config is translatable".
> If this is the case I will have one field to fill in django admin for each active langauge and I will be able to fill different values for different languages.
> The bottom line is that I will have the correct cookiespolicy language shown in my website based on the active language.
> I hope to have been clear in my messy explanation ;-)

`CONSTANCE_CONFIG = {
    'TITLE_INDEX': ('Главная', 'title'),
    'MAIN_1': (
        'Мы - проект и занимаемся нашим делом уже очень давномы производим то-то то-то уже в течении 032035 лет и мы '
        'лучшие на рынке среди наших конкурентов. ',
        'Header text '),
    'MAIN_2': ('НАША ПРОДУКЦИЯ', 'Header button'),
    'MAIN_3': ('О нас', 'About us'),
    'MAIN_4': (
        'Мы - проект и занимаемся нашим делом уже очень давномы производим то-то то-то уже в течении 032035 лет и мы '
        'лучшие на рынке среди наших конкурентов.',
        'about us text'),}
CONSTANCE_CONFIG_FIELDSETS = {
    'MainPage': (
        'TITLE_INDEX',
        'MAIN_1', 'MAIN_2', 'MAIN_3', 'MAIN_4', 'MAIN_5', 'MAIN_6', 'MAIN_7', 'MAIN_8', 'MAIN_9',
        'MAIN_10', 'MAIN_11',
        'MAIN_12', 'MAIN_13', 'MAIN_14', 'MAIN_15',),}
` 
excuse me. I have that now. How to translate this. I didn't quite understand what you were saying

**Comment by parruc** [comment]
Yes, I mean: if in my site I have Russian and English enabled I would like to be able to have, for the constance fields that are marked as translatable, 2 fields in the admin area: MAIN_1_RU and MAIN_1_EN so I can give 2 different texts for the two languages.

The same that happens in the django-modeltranslation package.
